### ISSUE TITLE
Typo in beta_binomial_model.py?

### ISSUE THREAD

**Issue by stephan-hutter** [smshuai/U1-snRNA#1]
Line 191 looks like this:
`Ns_n = normal_panel.loc[(row.chrom, row.pos, row.alt), 'dpP'].values`

I assume you are looking at total depth on the negative strand. Shouldn't the `'dpP'` then be `'dpN'`?

**Comment by smshuai** [comment]
You are right about this. This shall affect both normal and tumour scores in the negative strand the same way. I need to check what exactly is affected. Thanks again for pointing this out!
